.github/workflows/backend-api-spec-pr.yml

```diff
@@ -0,0 +1,28 @@
+name: Create PR for new version of backend api spec
+on:
+  schedule:
+    # triggers the workflow every day at 5:45 UTC
+    - cron: '45 5 * * *'
+
+# FixMe: Properly should be:
+#  push:
+#    branches:
+#      - 'infra/update-backend-api-spec'
```

Comment:
For some reasons, when bot update this branch via `backend-api-spec-update.yml` (another action), current job weren't trigger
its triggers only after manual push

so in this file I used schedule too 😐

Reply:
maybe combine these two action to one?

Reply:
good point, yes. probably we will not soon investigate this problem, so better solution will be to combine them

Reply:
Fixed

Synthetically triggered action created PR successfully https://github.com/saveourtool/save-cloud/pull/2038